operator/go.mod

```diff
@@ -6,21 +6,16 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.2 // indirect
 	github.com/Azure/go-autorest/autorest/adal v0.7.0 // indirect
 	github.com/go-logr/logr v0.1.0
-	github.com/gogo/protobuf v1.2.1 // indirect
```

Comment:
Should we revert the changes to the `go.mod` and `go.sum` dependencies of the `operator`? What triggered these?

Reply:
Have merged with master and run `go mod tidy`